CHANGES.rst

```diff
@@ -13,6 +13,29 @@ Changelog
 
 .. towncrier release notes start
 
+
+3.1.0 (2020-02-03)
+==================
+
+
+Features
+--------
+
+- Advisory now support reboot_suggested info.
+  `#5737 <https://pulp.plan.io/issues/5737>`_
+- Skip unsupported repodata.
+  `#6034 <https://pulp.plan.io/issues/6034>`_
```

Comment:
This file is still in the CHANGES dir, is that intentional?

Reply:
It's not, thanks! Removed.